mcode/abstraction-tool
# Update QuestionnaireResponse Upon Input Change

Created a button that allows the user to generate a QeustionnaireResponse after form modifications have been made. Clicking the "Generate Questionnaire Response" button should:
1) Read the rendered form in the HTML element and generate a corresponding QuestionnaireResponse
2) Log the questionnaire to the console
3) Print a fun little message below the button

**To Test:**
I modified the `sample-patient-bundle` so that two CancerRelatedSurgicalProcedures exist. Clicking the "Generate Questionnaire Response" button without selecting any answer options should console.log() a questionnaire response with only one item containing an answer. If you then choose one of the answer options for CancerRelatedSurgicalProcedure, and re-click the button, the console should log a new questionnaire response containing two items with answers, since you have chosen a CancerRelatedSurgicalProcedure answer.

*Note: I left the `sample-patient-bundle` modifications here so that it can be tested, but I can delete the extra resource after this is reviewed.*
